server/src/main/java/com/exacaster/lighter/application/sessions/SessionHandler.java

```diff
@@ -73,6 +78,72 @@ public void keepPermanentSessions() throws InterruptedException {
         LOG.info("End provisioning permanent sessions.");
     }
 
+    @SchedulerLock(name = "keepPermanentSession", lockAtLeastFor = "1m")
+    @Scheduled(fixedRate = "1m", initialDelay = "2s")
```

Comment:
I had to add initialDelay as it interfere with "handles soft deletes" test :(((((

Reply:
do we need both `keepPermanentSessions` & `keepPermanentSessions2`, looks like they are overlapping

Reply:
nope. Will keep just one. Nevertheless micronaut from testing perspective is really bad if one can not easily disable schedulng or do proper test slices

Reply:
only 1 keepPermanentSessions